lnwallet/channel.go

```diff
@@ -2398,16 +2398,6 @@ func NewBreachRetribution(chanState *channeldb.OpenChannel, stateNum uint64,
 	// remote commitment transaction.
 	htlcRetributions := make([]HtlcRetribution, 0, len(revokedSnapshot.Htlcs))
 	for _, htlc := range revokedSnapshot.Htlcs {
-		// If the HTLC is dust, then we'll skip it as it doesn't have
-		// an output on the commitment transaction.
-		if HtlcIsDust(
```

Comment:
Have an easy pointer just to confirm this?

Reply:
I think we [skip adding the htlc if dust](https://github.com/lightningnetwork/lnd/blob/master/lnwallet/commitment.go#L647)?

Reply:
That's to the commit tx, but this `Htlcs` stores all htlcs incl dust

Reply:
ah yeah you are right!  Will delete this commit!